common/openapi.yml

```diff
@@ -4,6 +4,24 @@ info:
   description: A manager for external credentials
   version: 0.0.1
 paths:
+  /api/oauth/v1/{provider}/authorization-url:
+    parameters:
+      - $ref: '#/components/parameters/providerParam'
+      - $ref: '#/components/parameters/redirectUriParam'
```

Comment:
I think we need to do a bit more describing on the parameters here. The client generation needs to know if its a parameter in the path, query, etc. Take a look at https://swagger.io/docs/specification/describing-parameters/.

Looks like it wasn't set up correctly in the first place, so lets take this opportunity to make sure the new endpoints are properly described.

Reply:
Oh interesting! Thanks for catching that

Reply:
Actually it looks right to me 🤔 The additional info you mentioned is defined here:
- https://github.com/DataBiosphere/terra-external-credentials-manager/blob/091a8fc03204f4bd6b12893ffb792226143b7808/common/openapi.yml#L268
- https://github.com/DataBiosphere/terra-external-credentials-manager/blob/091a8fc03204f4bd6b12893ffb792226143b7808/common/openapi.yml#L274